services/slack/intermediate.go

```diff
@@ -493,7 +466,7 @@ func (t *Transformer) AddAttachmentsToPost(post *SlackPost, newPost *Intermediat
 	return props, propsByteArray
 }
 
-//func (t *Transformer) TransformPosts(slackExport *SlackExport, attachmentsDir string, skipAttachments, discardInvalidProps bool) error {
+// func (t *Transformer) TransformPosts(slackExport *SlackExport, attachmentsDir string, skipAttachments, discardInvalidProps bool) error {
```

Comment:
What's this comment? :thinking:

Reply:
I have no idea. My `gofmt` changes this automatically. I think it should be removed completely.